cli/src/pcluster/templates/cdk_builder_utils.py

```diff
@@ -549,7 +549,7 @@ def _build_policy(self) -> List[iam.PolicyStatement]:
             iam.PolicyStatement(
                 sid="ResourcesS3Bucket",
                 effect=iam.Effect.ALLOW,
-                actions=["s3:*"],
+                actions=["s3:GetObject", "s3:ListBucket"],
```

Comment:
Is this enough? In cookbook I see some calls to `aws s3 presign` and some `aws s3api get-object` with `--version-id`

Reply:
Also do we need to update the template in tests?

Reply:
I searched in cookbook and node but I missed to look on user-data, thanks!

Reply:
In the templates they are already scoped down.
``` 
          PolicyName: HeadNodeRoleSlurmPolicy
          PolicyDocument:
            Statement:
              - Action:
                  - s3:GetObject
                  - s3:GetObjectVersion
                Effect: Allow
                Resource:
                  - !Sub arn:${AWS::Partition}:s3:::${AWS::Region}-aws-parallelcluster/*
                  - !Sub arn:${AWS::Partition}:s3:::dcv-license.${AWS::Region}/*
                  - !Sub arn:${AWS::Partition}:s3:::parallelcluster-*-v1-do-not-delete/*
```
and
```
          PolicyName: HeadNodeRoleBatchPolicy
          PolicyDocument:
            Statement:
              - Action:
                  - s3:GetObject
                  - s3:PutObject
                  - s3:GetObjectVersion
                Effect: Allow
                Resource:
                  - !Sub arn:${AWS::Partition}:s3:::parallelcluster-*-v1-do-not-delete/*
```

There is no `s3:*` in the `cluster-roles.cfn.yaml`.